tabs.svelte

```diff
@@ -0,0 +1,44 @@
+<script>
+  let items = [
+    { label: 'TAB 1', active: false },
+    { label: 'TAB 2', active: false },
+    { label: 'TAB 3', active: true },
+    { label: 'TAB 4', active: false },
+  ];
+</script>
+
+<style>
+  .content {
+    background-color: #fff;
+    width: 100%;
+    display: block;
+    justify-content: space-between;
+  }
+  .tab {
+    display: flex;
+    height: 48px;
+    justify-content: center;
+    align-items: center;
+    font-size: 14px;
+    font-weight: 600;
+    letter-spacing: 0.5px;
+    color: #8c8c8c;
+  }
+  .active {
+    color: #624695;
+  }
+  .line {
+    background-color: #624695;
+    height: 2px;
+    width: 100%;
+  }
+</style>
+
+{#each items as item}
+  <div class="content">
+    <div class="tab {item.active ? 'active' : ''}">{item.label}</div>
```

Comment:
You can use `class:` directive

eg.: `class:item.active`

Reply:
But, it is only a base component, not the final structure. but sorry :)